components/lib/multiselect/MultiSelect.js

```diff
@@ -158,7 +158,7 @@ export const MultiSelect = React.memo(
         };
 
         const onClick = (event) => {
-            if (!props.inline && !props.disabled && !props.loading && !isPanelClicked(event) && DomHandler.getAttribute(event.target, 'data-pc-section') !== 'removetokenicon' && !isClearClicked(event)) {
+            if (!props.inline && !props.disabled && !props.loading && !isPanelClicked(event) && !isClearClicked(event)) {
```

Comment:
Are you sure this change is necessary this looks like its there for unstyled mode?  I think the change below is all you need?

Reply:
Yep, since we are calling `event.stopPropagation`, the code handling this event won't be executed at all, regardless of whether we pass `unstyled` or not.